src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -27,6 +27,76 @@ public sealed partial class ApplicationIdentity : System.Runtime.Serialization.I
     }
 }
 
+namespace System.Data.Common
+{
+    public abstract partial class DBDataPermission : System.Security.CodeAccessPermission, System.Security.Permissions.IUnrestrictedPermission
+    {
+        [Obsolete]
```

Comment:
@jguady Yeah, Dan didn't notice that it was (msg, true), he thought it was just (msg).  And since there's no true-without-msg the msg needs to come back (or the member should be deleted)